rdt-open-reader/src/test/java/io/ona/rdt/robolectric/util/UtilsTest.java

```diff
@@ -223,4 +230,43 @@ public void testIsValidJSONObjectShouldReturnCorrectStatus() {
         Assert.assertFalse(Utils.isValidJSONObject(null));
         Assert.assertFalse(Utils.isValidJSONObject(""));
     }
+
+    @Test
+    public void testVerifyUserAuthorizationShouldCallOnlyOnce() {
```

Comment:
Add another testcase for `RUNNING` state

Reply:
I have tried to get RUNNING state but it returns FINISH state because I'm using SyncUtilShadow

Reply:
you can make the shadow return anything you want, provide setters and getters to modify state

Reply:
verify running state in the same test case